docs/sources/enterprise/access-control/concepts/permissions.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "Permissions"
+description = "Understand permissions in fine-grained access control"
+keywords = ["grafana", "fine-grained access-control", "concepts", "roles", "permissions", "enterprise"]
+weight = 130
++++
+
+# Permissions 
```

Comment:
It is not applicable at all to OSS Grafana from usage and product perspective. That said, a lot of code is in the OSS Grafana, so it might be valuable to cover some parts of this in a developer guide. That would be out of scope of this PR though.